### DISCUSSION TITLE
Load cached result after authentication

### DISCUSSION THREAD

**Discussion by wpfilf** [strapi-community/strapi-plugin-rest-cache#43] [Q&A] [answered]
I was wondering if it would be possible to cache results for private routes as well. Is it somehow possible to run the cache after the user-permissions plugin?

kind regards
Will

**Comment by stafyniaksacha** [comment, marked as answer]
Hello @wpfilf

I think you are looking the user cache example here: https://strapi-community.github.io/strapi-plugin-rest-cache/guide/strategy/cache-keys.html

**Reply by wpfilf** [reply]
Yes! That's what I was missing. Thank you so much!!
